spp_import_dci_api/models/group.py

```diff
@@ -0,0 +1,60 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class OpenSPPGroup(models.Model):
+    _inherit = "res.partner"
+
+    grp_is_created_from_crvs = fields.Boolean("Imported from CRVS", store=True)
+
+    no_of_child_under_12_months = fields.Integer(
+        "Number of child under 12 months",
+        compute="_compute_child_under_no_of_months",
+        store=True,
+    )
+
+    child_under_12_birthplace = fields.Text(
+        "Child under 12 months birthplace",
+        compute="_compute_child_under_no_of_months",
+        store=True,
+    )
+
+    @api.depends("group_membership_ids")
```

Comment:
I may post this comment on another commit, but I doubt this will re-calculate the number of children under 12 months, like this year you come into form view for this record, it's 2, next year, it still 2, if no changes happened to this o2m relationship model records